docs/user/welcome/upgrade.rst

```diff
@@ -29,139 +29,167 @@ But first to upgrade - change your dependency to |release| (or an appropriate st
+
+Using the command line to update your own pom.xml files::
+
+   git grep -l com.vividsolutions | grep pom.xml | xargs sed -i "s/com.vividsolutions/org.locationtech.jts/g"
+   
+And codebase::
+
+   git grep -l com.vividsolutions | xargs sed -i "s/com.vividsolutions/org.locationtech/
+
+**Use of copy rather than clone**
+
+If you are in the habit of using **clone** to duplicate JTS objects (such as Geometry and Coordinate) you will find the **clone** method has been deprecated, and a **copy** method introduced to explicitly perform a deep copy::
+    
+    Geometry duplicate = geom.copy();
+
+Migrate to JSR-363 Units
+^^^^^^^^^^^^^^^^^^^^^^^^
+
 This releases upgrades from the unofficial JSR-275 units library to the official JSR-363 units API.
 
-* Maven transitive dependency will correctly bring in the required jars::
+Maven transitive dependency will correctly bring in the required jars::
    
     <dependency>
        <groupId>systems.uom</groupId>
        <artifactId>systems-common-java8</artifactId>
        <version>0.7.2</version>
     </dependency>
 
-* Package names have changed, resulting in some common search and replaces when upgrading:
+Package names have changed, resulting in some common search and replaces when upgrading:
   
-  * Search ``javax.measure.unit.Unit`` replace ``javax.measure.Unit``
-  * Search ``ConversionException`` replace  ``IncommensurableException``
-    
-    This is a checked exception, in areas of the GeoTools library where this was found we now return an IllegalArgument exception.
-    
-  * Search ``converter == UnitConverter.IDENTITY`` replace ``converter.isIdentity()``
-  * Search ``javax.measure.unit.NonSI`` replace ``import si.uom.NonSI``
-  * Search ``javax.measure.unit.SI`` replace ``import si.uom.SI``
-  * Search ``SI.METER`` replace ``SI.METRE``
-  * Search ``javax.measure.converter.UnitConverter`` replace ``javax.measure.UnitConverter``
-  * Search ``javax.measure.unit.UnitFormat`` replace ``import javax.measure.format.UnitFormat``
-  * Search ``Unit.ONE`` replace ``AbstractUnit.ONE``
-  * Search ``Dimensionless.UNIT`` replace ``AbstractUnit.ONE``
-  * Search ``Unit.valueOf(unitString)`` replace ``Units.parseUnit(unitString)``
+* Search ``javax.measure.unit.Unit`` replace ``javax.measure.Unit``
+* Search ``ConversionException`` replace  ``IncommensurableException``
+  
+  This is a checked exception, in areas of the GeoTools library where this was found we now return an IllegalArgument exception.
   
-* Getting Unit instances
+* Search ``converter == UnitConverter.IDENTITY`` replace ``converter.isIdentity()``
+* Search ``javax.measure.unit.NonSI`` replace ``import si.uom.NonSI``
+* Search ``javax.measure.unit.SI`` replace ``import si.uom.SI``
+* Search ``SI.METER`` replace ``SI.METRE``
+* Search ``javax.measure.converter.UnitConverter`` replace ``javax.measure.UnitConverter``
+* Search ``javax.measure.unit.UnitFormat`` replace ``import javax.measure.format.UnitFormat``
```

Comment:
Does JSR-363 have both si.uom.* and javax.measure.* packages?

Reply:
jar-363 has javax.measure.* packages.

You need to choose your own implementation - we chose si.uom.*